Kernel/ParallelTransport.m

```diff
@@ -245,11 +297,12 @@
 method = OptionValue["Method"];
 param = OptionValue["Parametrization"];
 
-If[param =!= "Mino", Print["Only Mino time parametrization has been implemented for parallel transport."];Return[];];
+If[param =!= "Mino"&& param =!= "Phases", Print["Only Mino time parametrization has been implemented for parallel transport."];Return[];];
```

Comment:
Another instance of a Print statement that we probably want to change to a Message